### PR TITLE
Set up webpack and initial settings of ToDo List

### PR DESCRIPTION
**Pull request by M-Anwar-Hussaini**
# New🆕 Features In This Milestone🥇:
- [x] 📌Set up webpack.✅
- [x] 📌Meet the current phase [requrements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md).✅
- [x] 📌Build the general structure of TO-DO list project.✅
- [x] 📌Use BootTrap to style the page.✅
- [x] 📌No linters error.✅
- [x] 📌Follow Github workflow.✅
- [x] 📌Professional README file.✅

## To the Code Reviewer:
**Thanks a lot for dedicating your time to reviewing this PR**
- [ ] _I kindly ask 🙏 dear code reviewer 🤵 that if there is any kind of issue 🦯 in this project, please do list 📃 them in a descriptive 💡 manner and give your best suggestions 🎁 if needed._

- [ ] _If you think there is some big issues 👿 that is essential to be changed ♻️  please kindly contact 📞 me through **Zoom** or **Slack**, I'm mostly available.__

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  },
+  {
+    description: 'Go to college',
+    completed: true,
+  },
+  {
+    description: 'Request for a review',
+    completed: false,
+  },
+];
```
Comment (Strangeal) [inline src/index.js]: - [ ] As specified in the project requirement kindly consider adding an `index` for each object of the array such that you can identify each todo task by its id and can also remove tasks in the future when you implement that functionality. See below for more details:
Screenshot 2023-06-07 at 07 03 36

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+const displayTasks = () => {
+  const elTaskList = document.querySelector('.all-tasks');
+  elTaskList.innerHTML = '';
+  allTasks.forEach((el) => {
+    // Create a task
+    const task = document.createElement('li');
+    task.className = 'd-flex gap-2 mx-2 pb-1 align-items-center border-bottom border-secondary-subtle ';
```
Comment (Strangeal) [inline src/index.js]: - [ ] Kindly consider rendering each todo task based on their `index` as specified in the project requirement. See below for more details:
Screenshot 2023-06-07 at 07 03 24

**Review by Strangeal — CHANGES_REQUESTED** [review]
Hi @M-Anwar-Hussaini, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text

## Highlights
Correct use of GitHub Flow ✅
Well-structured files ✅
All linter checks are passing ✅
Descriptive PR title with a great summary ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @M-Anwar-Hussaini, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

<div align='center'>

</div>

## Good Highlights✔️

- Requested changes have been implemented

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**